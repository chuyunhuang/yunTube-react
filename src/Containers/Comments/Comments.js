import React from 'react';
import { Button, Icon } from 'semantic-ui-react';
import './CommentHeader.scss';

export function CommentHeader(props) {
  return (
    <div className="comments-header">
      <h4>Comments</h4>
      <Button basic compact icon labelPosition="left">
        <Icon name="align-left" />
      </Button>
    </div>
  )
}