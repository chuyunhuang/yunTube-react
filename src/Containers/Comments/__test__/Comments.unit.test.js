import React from 'react';
import { shallow } from 'enzyme';
import { Comments } from '../Comments';

describe('Comments', () => {
  test('renders Comments w/o props', () => {
    const wrapper = shallow(<Comments />)
    expect(wrapper).toMatchSnapshot();
  });

  test('renders Comments with props.amountComments={123}', () => {
    const wrapper = shallow(<Comments amountComments={123} />)
    expect(wrapper).toMatchSnapshot();
  })

})
