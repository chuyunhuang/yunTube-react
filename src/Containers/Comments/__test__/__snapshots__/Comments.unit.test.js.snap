// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders Comments w/o props 1`] = `
<div>
  <CommentHeader />
  <AddComment />
  <SingleComment />
  <SingleComment />
  <SingleComment />
</div>
`;

exports[`Comments renders Comments with props.amountComments={123} 1`] = `
<div>
  <CommentHeader
    amountComments={123}
  />
  <AddComment />
  <SingleComment />
  <SingleComment />
  <SingleComment />
</div>
`;
